v2/AdminCore.Services.Tests/EventServiceTests.cs

```diff
@@ -45,7 +42,7 @@ public void CreateEvent_WhenCalled_SuccessfullyInsertsNewEventIntoDb()
       };
 
       // Act
-      _eventService.CreateEvent(eventDateDto, EventTypes.AnnualLeave);
+      _eventService.CreateEvent(eventDateDto, EventTypes.AnnualLeave, 1);
```

Comment:
Create a local variable for this employeeId

Reply:
Fixed